phpunit/class-rest-shortcodes-controller-test.php

```diff
@@ -137,46 +131,16 @@ public function data_get_item_with_field_combinations() {
 					'postId'    => self::$post_id,
 				),
 				array(
-					'html'  => 'Sorry, couldn\'t render a preview',
-					'type'  => 'html',
-					'style' => '',
-					'js'    => '',
-				),
-			),
-			// Youtube embed.
```

Comment:
this shouldn't be removed. Focus on testing the html (perhaps not as part of this providing set)